### PR TITLE
Add scripts for midas migration

### PR DESCRIPTION
**Pull request by fogleman**
@zachmullen @cpatrick @mgrauer

### CODE REVIEW

**Comment by zachmullen** [comment]
Fixes #14

**Comment by mgrauer** [comment]
LGTM

**Comment by zachmullen** [comment]
Hang on, can we add a README mentioning the prerequisites to install, and also mentioning that several values near the top of the script need to be replaced for it to work?

**Comment by cjh1** [comment]
Is it worth adding a requirements.txt to make it easy for a user to install the deps?

File: scripts/midas/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+girder-client
+joblib
+pydas
```
Comment (zachmullen) [inline scripts/midas/requirements.txt]: What about sqlite?

Comment (cjh1) [inline scripts/midas/requirements.txt]: Its in the standard lib I think

Comment (fogleman) [inline scripts/midas/requirements.txt]: It is.

Comment (zachmullen) [inline scripts/midas/requirements.txt]: :mind_blown:

Comment (fogleman) [inline scripts/midas/requirements.txt]: sqlite is awesome.

**Comment by zachmullen** [comment]
Cool, LGTM

**Pull request merged**